detectron2/evaluation/coco_evaluation.py

```diff
@@ -49,6 +50,14 @@ def __init__(self, dataset_name, cfg, distributed, output_dir=None):
         self._logger = logging.getLogger(__name__)
 
         self._metadata = MetadataCatalog.get(dataset_name)
+
```

Comment:
The docs at line 39 should be updated.

It must have the following ....; Otherwise, it must be in detectron2's standard dataset format so it can be converted to COCO format automatically.

Reply:
Done